bears/vcs/git/GitCommitBear.py

```diff
@@ -109,6 +113,37 @@ def check_shortlog(self,
                 yield Result(
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
+        if shortlog_imperative_check:
+            colon_pos = shortlog.find(':')
+            shortlog = shortlog[colon_pos:] if colon_pos != -1 else shortlog
```

Comment:
`shortlog[colon_pos:]` would also extract the colon. Shouldn't it be `shortlog[colon_pos + 1:]`?